src/Rx.ts

```diff
@@ -65,6 +65,7 @@ import './add/operator/mergeMap';
 import './add/operator/mergeMapTo';
 import './add/operator/multicast';
 import './add/operator/observeOn';
+import './add/operator/pairwise';
```

Comment:
This would not be required if this operator's placed under `kitchenSink`, since importing `add` will patch prototype immediately.